.github/workflows/ci.yml

```diff
@@ -29,6 +29,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
+      # Can be removed once https://github.com/actions/runner-images/pull/13076 is released
```

Comment:
Let's remove this to make the diff clear.
I've pushed the recent fixes from `master` to `0.11.x`